packages/core/admin/server/services/token.js

```diff
@@ -49,9 +49,22 @@ const decodeJwtToken = token => {
   }
 };
 
+/**
+ * @returns {void}
+ */
+const checkSecretIsDefined = () => {
+  if (strapi.config.serveAdminPanel && !strapi.config.get('admin.auth.secret')) {
+    const secretExample = crypto.randomBytes(16).toString('base64');
+    throw new Error(
+      `Missing auth.secret. Please set auth.secret in config/admin.js (ex: ${secretExample})`
```

Comment:
We shouldn't tell them to statically set the key in this file, they should use the environment variable.